src/IQFeed.CSharpApiClient/Lookup/Chains/ChainsRequestFormatter.cs

```diff
@@ -20,10 +20,12 @@ public string ReqChainFutureOption(string symbol, OptionSideFilterType optionSid
             return request;
         }
 
+        // Protocol Update to 6.1 - Added "includeNonStandardOptions" - IQ Default is false
         public string ReqChainIndexEquityOption(string symbol, OptionSideFilterType optionSideFilter, string monthCodes, int? nearMonths = null, BinaryOptionFilterType binaryOptionFilter = BinaryOptionFilterType.Include,
-            OptionFilterType optionFilter = OptionFilterType.None, int? filterValue1 = null, int? filterValue2 = null, string requestId = null)
+            OptionFilterType optionFilter = OptionFilterType.None, int? filterValue1 = null, int? filterValue2 = null, string requestId = null, bool includeNonStandardOptions = false)
         {
-            var request = $"CEO,{symbol},{optionSideFilter.ToString().ToLower()},{monthCodes},{nearMonths},{(int)binaryOptionFilter},{(int)optionFilter},{filterValue1},{filterValue2},{requestId}{IQFeedDefault.ProtocolTerminatingCharacters}";
+            var wireFormatIncludeNonStandardOptions = includeNonStandardOptions ? 1 : 0;
+            var request = $"CEO,{symbol},{optionSideFilter.ToString().ToLower()},{monthCodes},{nearMonths},{(int)binaryOptionFilter},{(int)optionFilter},{filterValue1},{filterValue2},{requestId},{wireFormatIncludeNonStandardOptions}{IQFeedDefault.ProtocolTerminatingCharacters}";
```

Comment:
I think `binaryOptionFilter` doesn't exist anymore. I don't see it in docs at least. Moreover, I didn't tried your branch, but I changed protocol to 6.1 ( in master code) and added `includeNonStandardOptions` and got syntax error response. After deleting `binaryOptionFilter` the error disappeared.

Reply:
@sharok thanks